moveit_planners/stomp/include/stomp_moveit/math/multivariate_gaussian.hpp

```diff
@@ -91,16 +87,15 @@ MultivariateGaussian::MultivariateGaussian(const Eigen::MatrixBase<Derived1>& me
                                            const Eigen::MatrixBase<Derived2>& covariance)
   : mean_(mean), covariance_(covariance), covariance_cholesky_(covariance_.llt().matrixL()), normal_dist_(0.0, 1.0)
 {
-  rng_.seed(rand());
+  rng_.seed(rd_());
```

Comment:
I missed in your original PR that the rng is seeded and not just the default constructor used :see_no_evil: 
```suggestion
```

Reply:
It was seeded with rand(), but that wasn't generating random sequences either, as I checked at your suggestion, so I changed it to random_device. But either way, the RSL implementation looks a lot cleaner.

Is there any issue/PR reference explaining how it is more performant? Cause from what I see in the above links, rsl::rng() internally uses random_device.

Reply:
Performance issues are mainly caused by creating multiple random devices (e.g. if you construct a lot of objects that have a random device member). Using the RSL implementation improves that by using a thread_local generator and only creating a new one (and a random device instance) if it does not exist within the a thread. Otherwise the already existing instance is shared.